apps/server/src/modules/files-storage/entity/filerecord.entity.ts

```diff
@@ -1,8 +1,10 @@
 import { Embeddable, Embedded, Entity, Enum, Index, Property } from '@mikro-orm/core';
 import { ObjectId } from '@mikro-orm/mongodb';
+import { InternalServerErrorException } from '@nestjs/common';
 import { v4 as uuid } from 'uuid';
-import type { EntityId } from '../types/entity-id';
-import { BaseEntity } from './base.entity';
+import { BaseEntity } from '../../../shared/domain/entity/base.entity';
+import type { EntityId } from '../../../shared/domain/types/entity-id';
```

Comment:
from '@shared/domain' ?

Reply:
We have a dependency cycle via all-entities.ts

Reply:
Removed files storage entities from ALL_ENTITIES